src/main/java/org/dataone/cn/indexer/solrhttp/SolrDoc.java

```diff
@@ -28,7 +28,7 @@
 import java.util.List;
 
 import org.apache.commons.io.IOUtils;
-import org.dataone.service.types.v2.SystemMetadata;
+import org.dataone.service.types.v1.SystemMetadata;
```

Comment:
If the system metadata file is the v1 type, it would not work to use the v2 SystemMetadata object to represent it. The parent class can cover the child class. But the child class can't cover the parent class.